src/commands/publish/mod.rs

```diff
@@ -109,8 +107,8 @@ fn publish_script(
     }
 
     let pattern = if target.route.is_some() {
-        let route = Route::new(&target)?;
-        Route::publish(&user, &target, &route)?;
+        let route = route::Route::new(&target)?;
+        route::publish(&user, &target, &route)?;
```

Comment:
maybe out of scope for this but perhaps we should have this be `route.publish` instead of `route::publish` - since we already pass `&target` to `Route::new`, it seems strange to _also_ pass it to `route::publish`

Reply:
in the next PR i am introducing a method to `target`, `routes` that returns a vec of all routes that we will iterate over, so this function will no longer take a route as a parameter, and the Route struct will become a function-less data structure.

Reply:
renamed this "publish_route"